.github/workflows/release.yml

```
@@ -54,13 +54,27 @@
+        if: steps.skip_check.outputs.should_skip != 'true'
+        id: generate_notes
+        shell: pwsh
+        run: |
+          $PREV_TAG = "${{ steps.prev_tag.outputs.prevtag }}"
+          $CURRENT_TAG = $env:GITHUB_REF.Split('/')[-1]
+          echo "## Changes since $PREV_TAG" > RELEASE_NOTES.md
+          git log $PREV_TAG..$CURRENT_TAG --pretty=format:"- %s ([%h](%H))" >> RELEASE_NOTES.md
+
+      - name: DEBUG show release notes
+        if: steps.skip_check.outputs.should_skip != 'true'
+         run: type .\RELEASE_NOTES.md
```

actionlint

could not parse as YAML: yaml: line 71: mapping values are not allowed in this context

(syntax-check)